pyrasterframes/src/main/python/tests/PyRasterFramesTests.py

```diff
@@ -574,7 +574,7 @@ def test_matmul(self):
         nd = r1.cell_type.no_data_value()
         e1 = Tile(np.ma.masked_equal(np.array([[nd, 10],
                                                [nd, nd]], dtype=r1.cell_type.to_numpy_dtype()), nd))
-        self.assertTrue(r1 == e1)
+        self.assertEqual(r1, e1)
```

Comment:
I haven't figured out why yet, but `test_matmul` is failing:

```
>       self.assertEqual(r1, e1)
E       AssertionError: Tile([18 chars]ta=[[ 7, 10],
E               [15, 22]],
E         mask=False,[41 chars]9999) != Tile([18 chars]ta=[[--, 10],
E               [--, --]],
E         mask=[[ Tru[72 chars]ud63)

tests/PyRasterFramesTests.py:577: AssertionError
```

Might have happened after I merged `develop` onto `feature/pip-installable`.